infra/modular-examples/aws-google-workspace/main.tf

```diff
@@ -25,8 +25,9 @@ module "worklytics_connector_specs" {
 
   enabled_connectors             = var.enabled_connectors
   google_workspace_example_user  = var.google_workspace_example_user
-  google_workspace_example_admin = coalesce(var.google_workspace_example_admin, var.google_workspace_example_user)
+  google_workspace_example_admin = try(coalesce(var.google_workspace_example_admin, var.google_workspace_example_user), null)
   salesforce_domain              = var.salesforce_domain
+  msft_tenant_id                 = var.msft_tenant_id
```

Comment:
this is the fix, pass through `msft_tenant_id` to `worklytics_connector_specs` module